fiftyone/server/db/mongo.py

```
@@ -0,0 +1,174 @@
+
+        from fiftyone.server.aggregations import _resolve_path_aggregation
+        from fiftyone.server.exceptions import AggregationQueryTimeout
+
+        aggregations, deserializers = zip(
+            *[
+                _resolve_path_aggregation(
+                    path, view, form.query_performance, form.hint
+                )
+                for path in form.paths
+            ]
+        )
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)